src/kube-runtime/src/runtime.d/user.sh

```diff
@@ -17,5 +17,8 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+set -o errexit
+#set -o nounset
```

Comment:
Why remove this?

Reply:
Currently remove this because the injected code may break this and cause error. Temporarily remove this.
Will enable this after change webportal code.

Reply:
Why runtime depends on webportal ?

Reply:
It's because webportal may inject some codes in user commands. Removed the codes that may cause error.

Reply:
Can you be specific?